beetmoverscript/src/beetmoverscript/constants.py

```diff
@@ -78,6 +78,8 @@
 
 RELEASE_ACTIONS = ("push-to-releases",)
 
+DIRECT_RELEASE_ACTIONS = ("direct-push-to-releases",)
```

Comment:
I'm not very happy about the way I named this function since we're going to reuse this for Fenix nightlies too. But after all it's just naming thing since the logic is dictated in the task payload. Though we may want to revisit and s/direct-push-to-releases/direct-push-to-bucket` or alike so make sure it's not confusing us in the future.